demo/Makefile

```diff
@@ -0,0 +1,22 @@
+IMAGE=cloudbees/github-organization-demo
+TAG=$(shell sed -ne s/cloudbees-github-pull-requests://p < plugins.txt)
+# an arbitrary directory we can share with the host
+WORKSPACE=$(shell pwd)/workspace
+
+build:
+    docker build -t $(IMAGE):$(TAG) .
+
+run: build
+    rm -rf $(WORKSPACE)
+    mkdir $(WORKSPACE)
+    docker run --rm -p 8080:8080 -p 4040:4040 -v /var/run/docker.sock:/var/run/docker.sock -v $(WORKSPACE):$(WORKSPACE) -e WORKSPACE=$(WORKSPACE) -ti $(IMAGE):$(TAG)
+
+# TODO support snapshots somehow TBD
```

Comment:
See [this](https://github.com/jenkinsci/parallel-test-executor-plugin/blob/52ee8cf5f8971aa7db44623a310d538b67073a54/demo/Makefile#L11-L18) as an example.